README.md

```diff
@@ -1,6 +1,6 @@
 # razzle-plugin-workbox
 
-A [Razzle](https://github.com/jaredpalmer/razzle) plugin for seting up [workbox](https://developers.google.com/web/tools/workbox/modules/workbox-webpack-plugin) and register the service worker as [create react app](https://github.com/facebook/create-react-app) example [does](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/src/serviceWorker.js).
+A [Razzle](https://github.com/jaredpalmer/razzle) plugin for seting up [workbox](https://developer.chrome.com/docs/workbox/reference/workbox-webpack-plugin/) and register the service worker as [create react app](https://github.com/facebook/create-react-app) example [does](https://create-react-app.dev/docs/making-a-progressive-web-app/).
```

Comment:
The default [cra-templates](https://create-react-app.dev/docs/custom-templates/#finding-custom-templates) do not seem to include the `serviceWorker.js` file anymore? 

It might only exists in [community cra-templates](https://www.npmjs.com/search?q=cra-template-*) now (such as https://www.npmjs.com/package/cra-template-pwa-typescript)? 

I'm not sure, so I used the primary PWA documentation url for now.